### DISCUSSION TITLE
Разделение (смешанный) доступ к циклам с именением цвета

### DISCUSSION THREAD

**Discussion by IrinaG1090** [totumonline/totum-ru-issues-and-discussions#441] [Как это сделать?] [answered]
https://docs.totum.online/how-to-configure-loop-accesses-for-users

вопрос вот в чем: есть таблица циклов. есть разделение доступа как в этом обучающем ролике на admin и user1. Но теперь задача другая: если admin изменил доступ к циклу для User1 и этот цикл теперь видит и User1. То как "назначенный" только что цикл пометить цветом, что он - новый цикл для User1 среди других циклов принадлежащих только User1 (видит только свои циклы)? новый цикл может затеряться среди других циклов User1. Как его можно пометить цветом или жирностью для User1 когда меняется автор цикла? Может есть несколько путей? подскажите кто как видит, плиз.

**Comment by erpic-ru** [comment, marked as answer]
Добавить в расчетную в цикле системное поле new с чекбоксом и вывести его ярлыком в таблицу циклов. 
В таблице циклов поставить форматирование по условию на true и роль пользователя неадмин
Ставить true кодом действия при изменении поля с доступом. 
При заходе в цикл проверять пользователя и ставить кодом действия таблицы false, если true
